requirements/docs.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with python 3.8
+# This file is autogenerated by pip-compile with python 3.10
```

Comment:
Nit: why is the header updated to `Python 3.10` in `local.txt`, `docs.txt`, and `test.txt`, but not, e.g. for `production.txt`? I thought the `make upgrade` command would upgrade it for all files.
Using Python 3.10 with `make upgrade` could potentially update packages to versions that are no longer compatible with Python 3.8, which is fine for the CI, but not so fine for production requirements. We may want to be consistent here and use only Python 3.8 unless we test both versions in the CI.

Reply:
Whoops, my bad. I tried not to commit the "3.10" but missed a couple. And you're right, better to run it with 3.8 in the first place, so that's what I've done now. I don't think it changed anything besides a comment about which packages depend on `typing-extensions`